Resources/Prototypes/Nyanotrasen/Entities/Objects/Fun/pai.yml

```diff
@@ -0,0 +1,71 @@
+# Custom pAI types with different instruments
+
+# Making a nyanotrasen-specific copy of the entire pAI entity is necessary
+# because the base class's instrument component's bank and program must be unassigned
+# to allow for supersynth qualities. Without this, we would have to modify the original file.
```

Comment:
See space-wizards/RobustToolbox#2823 for more information on why exactly this is required. 
I can't just assign them null in a child of the PersonalAI entity, nulls do not get parsed properly.